cli/tests/activate.bats

```diff
@@ -3190,3 +3190,22 @@ EOF
   run "$FLOX_BIN" activate -- printenv FLOX_DISABLE_METRICS
   assert_output "true"
 }
+
+# ---------------------------------------------------------------------------- #
+
+@test "can use fallback interpreter" {
+  project_setup
+  run "$FLOX_BIN" activate --use-fallback-interpreter -- true
+  assert_success
+}
+
+@test "fallback flag activates with rendered interpreter" {
+  project_setup
+
+  # Attempting to use the interpreter bundled with the CLI will fail because
+  # we're overriding the variable for the bundled interpreter store path, so
+  # we should only be able to activate if the fallback interpreter is used.
+  FLOX_INTERPRETER="/foo" run "$FLOX_BIN" activate --use-fallback-interpreter -- true
+  assert_success
+}
+
```

Comment:
nonblocking: is it worth testing this in the other direction? If I activate an environment that has a bad interpreter in the rendered environment without `--use-fallback-interpreter`, does it activate correctly?

Reply:
I'm not actually sure how to get a bad interpreter into the environment since it's in the Nix Store.

Reply:
Not sure if it would break anything else but maybe `touch .flox/run/$system.$name/activate`